pytest.ini

```diff
@@ -3,6 +3,10 @@
 filterwarnings =
     ignore::DeprecationWarning
 
+markers =
+    nosetup_all: do not call setup_all 
```

Comment:
will this affect the entire codebase, or just when specifically asked for?

Reply:
> will this affect the entire codebase, or just when specifically asked for?

The latter. To invoke it for a given unit test, add decorator `@pytest.mark.nosetup_all` to the top of the unit test. (I will put that into the final squash-merged description too.)